client/my-sites/domains/domain-management/transfer/transfer-to-other-site/index.jsx

```diff
@@ -115,52 +113,59 @@ export class TransferToOtherSite extends React.Component {
 		}
 	};
 
-	render() {
-		if ( ! this.isDataReady() ) {
-			return <DomainMainPlaceholder goBack={ this.goToEdit } />;
+	getMessage() {
+		const { selectedDomainName: domainName, isMapping, translate } = this.props;
+		const translateArgs = { args: { domainName }, components: { strong: <strong /> } };
+
+		if ( isMapping ) {
+			return translate(
+				"Please choose a site you're an administrator on to transfer domain mapping for {{strong}}%(domainName)s{{/strong}} to:",
+				translateArgs
+			);
 		}
 
-		const { selectedSite, selectedDomainName, currentRoute } = this.props;
+		return translate(
+			"Please choose a site you're an administrator on to transfer {{strong}}%(domainName)s{{/strong}} to:",
+			translateArgs
+		);
+	}
+
+	render() {
+		const { selectedSite, selectedDomainName, currentRoute, isMapping } = this.props;
 		const { slug } = selectedSite;
+		if ( ! this.isDataReady() ) {
+			return (
+				<DomainMainPlaceholder
+					backHref={ domainManagementTransfer( slug, selectedDomainName, currentRoute ) }
+				/>
+			);
+		}
 
 		return (
 			<Main className="transfer-to-other-site">
 				<Header
 					selectedDomainName={ selectedDomainName }
 					backHref={ domainManagementTransfer( slug, selectedDomainName, currentRoute ) }
 				>
-					{ this.props.translate( 'Transfer Domain To Another Site' ) }
+					{ isMapping
+						? this.props.translate( 'Transfer Domain Mapping To Another Site' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.